### ISSUE TITLE
Make admin page titles more useful for admin users

### ISSUE THREAD

**Issue by timjmansfield** [wagtail/wagtail#9054]
### Is your proposal related to a problem?

Every single Wagtail admin screen puts “Wagtail” at the front (left) of the `<title>` – which occludes the more useful information of which page you might have open as soon as you have a few tabs.

The example tab I have open is the page editor for a page titled: “Organising A Museum Website”.

The page title is: "Wagtail - Editing News Article Page - Organising A Museum Website".

This is what I see:

<img width="193" alt="Screen Shot 2022-08-19 at 6 27 08 pm" src="https://user-images.githubusercontent.com/22737223/185579394-a8a907a6-9d9a-4334-ac3c-b0ff94385353.png">

If you have a few Wagtail admin tabs open they all say "Wagtail". This necessitates flipping between them (or at least hovering over every tab) to figure out which tab has which page.

### Describe the solution you'd like

For an experienced content editor, the favicon uniquely identifies Wagtail tabs, so for most pages, I think omitting "Wagtail" or shifting it to the end of the `<title>` would allow more salient information to be visible 

I think the most significant information presented in the `<title>` of a page editor tab is the title of the page being edited. The favicon tells you you're in the admin, so "Editing" is redundant.

Simply reordering the elements from most significant to least significant (from the perspective of a content editor) would improve the UI. To persist with my example:

`<title>Organising A Museum Website (News Article Page) - Editing - Wagtail</title>`

### Describe alternatives you've considered

I have not considered any alternatives.

### Additional context

I looked for related issues, but couldn't find any.

**Comment by lb-** [comment]
I think re-ordering the elements is a good idea. 

However, you can already override the word 'Wagtail' 

https://docs.wagtail.org/en/stable/advanced_topics/customisation/admin_templates.html#branding-title

**Comment by lb-** [comment]
Labelling accessibility. This would be a win for browser tab usability but also for better compliance.


### Links

- https://www.w3.org/WAI/WCAG21/Understanding/page-titled.html
- https://www.w3.org/WAI/WCAG21/Techniques/general/G88
- https://accessibility.iu.edu/creating-content/web-content/titles.html


### Approach proposal

https://github.com/wagtail/wagtail/blob/520dfa8005d8b11580dc91c4ff625afddf9c60ad/wagtail/admin/templates/wagtailadmin/skeleton.html#L8

```
        <title>{% block titletag %}{% endblock %} - {% block branding_title %}Wagtail{% endblock %} </title>
```

Should be an easy enhancement, labelling good first issue also.

**Comment by basic5253** [comment]
I would try this. In the admin screen, I would type the following
`def get_admin_display_title(self): #pulled from wagtail sourcecode in home models.py
return`

**Comment by basic5253** [comment]
> I would try this. In the admin screen, I would type the following `def get_admin_display_title(self): #pulled from wagtail sourcecode in home models.py return`

The return is going to be in another line and indented obv. I think I messed up the code in sending.

**Comment by lb-** [comment]
Resolved now - this is what an editing page looks like.

<img width="407" alt="Screen Shot 2022-08-31 at 10 04 11 pm" src="https://user-images.githubusercontent.com/1396140/187675438-9fc5234f-d0b0-4f73-a9aa-f3cc1351ded2.png">

If we want to further refine the layout (move the 'page type') out to after the title - best that we cover that in a new issue.
